platform/qt/src/qmapboxgl.cpp

```diff
@@ -657,19 +716,30 @@ void QMapboxGL::setPitch(double pitch_)
     d_ptr->mapObj->setPitch(pitch_);
 }
 
+/*!
+    Returns the North Orientation mode.
+*/
 QMapboxGL::NorthOrientation QMapboxGL::northOrientation() const
 {
     return static_cast<QMapboxGL::NorthOrientation>(d_ptr->mapObj->getNorthOrientation());
 }
 
+/*!
+    Sets the North Orientation mode to \a orientation.
+*/
 void QMapboxGL::setNorthOrientation(NorthOrientation orientation)
 {
     d_ptr->mapObj->setNorthOrientation(static_cast<mbgl::NorthOrientation>(orientation));
 }
 
-void QMapboxGL::setGestureInProgress(bool inProgress)
+/*!
+    Tells the map rendering engine that there is currently a gesture in \a progress. This
+    affects how the map renders labels, as it will use different texture filters if a gesture
```

Comment:
I think in this case it is fine, because this is the "Qt low-level API" that could be used for implementing higher level bindings like the QML component we are pushing to Qt 5.9.